modules/nw-multinetwork-sriov.adoc

```diff
@@ -40,247 +62,386 @@ and device ID 0x1015
-configured SR-IOV VFs (of supported NIC models) on each node and advertise
-discovered resources. The number of available SR-IOV VF resources that are
-capable of being allocated can be reviewed by describing a node with the
-[command]`oc describe node <node-name>` command. The resource name for the
-SR-IOV VF resources is `openshift.io/sriov`. When no SR-IOV VFs are available on
-the node, a value of zero is displayed.
+The SR-IOV Network Operator can also be install through the web console 
+following the directions below.
+
+Before that, you have to create the `Namespace` and `OperatorGroup` as mentioned
+in above section.
 ====
+
+.Procedure
+
+. Install the SR-IOV Network Operator using the {product-title} web console:
+
+.. In the {product-title} web console, click *Catalog* -> *OperatorHub*.
+
+.. Choose  *SR-IOV Network Operator* from the list of available Operators, and
+click *Install*.
+
+.. On the *Create Operator Subscription* page, under *A specific namespace on
+the cluster* select *sriov-network-operator*. Then, click *Subscribe*.
+
+. Verify the operator installations:
+
+.. Switch to the *Catalog* → *Installed Operators* page.
+
+.. Ensure that *SR-IOV Network Operator* is listed in the
+*riov-network-operator* project with a *Status* of *InstallSucceeded*.
+
 +
-[source,yaml,subs="attributes"]
-----
-kind: DaemonSet
-apiVersion: apps/v1
-metadata:
-  name: sriov-device-plugin
-  namespace: openshift-sriov
-  annotations:
-    kubernetes.io/description: |
-      This daemon set launches the SR-IOV network device plugin on each node.
-spec:
-  selector:
-    matchLabels:
-      app: sriov-device-plugin
-  updateStrategy:
-    type: RollingUpdate
-  template:
-    metadata:
-      labels:
-        app: sriov-device-plugin
-        component: network
-        type: infra
-        openshift.io/component: network
-    spec:
-      hostNetwork: true
-      nodeSelector:
-        kubernetes.io/os: linux
-      tolerations:
-      - operator: Exists
-      serviceAccountName: sriov-device-plugin
-      containers:
-      - name: sriov-device-plugin
-        image: quay.io/openshift/{image-prefix}-sriov-network-device-plugin:v4.0.0
-        args:
-        - --log-level=10
-        securityContext:
-          privileged: true
-        volumeMounts:
-        - name: devicesock
-          mountPath: /var/lib/kubelet/
-          readOnly: false
-        - name: net
-          mountPath: /sys/class/net
-          readOnly: true
-      volumes:
-        - name: devicesock
-          hostPath:
-            path: /var/lib/kubelet/
-        - name: net
-          hostPath:
-            path: /sys/class/net
-----
-
-. Run the following command to create the `sriov-device-plugin` DaemonSet:
+[NOTE]
+====
+During installation an operator might display a *Failed* status. If the operator
+then installs with an *InstallSucceeded* message, you can safely ignore
+the *Failed* message.
```

Comment:
Do we have an estimated time on how long the installation would take? perhaps give a number here so that user has expectation on how long they shall wait.

Reply:
It mainly depends on how long the operator images can be downloaded from Internet, which it hard to predict in customer's environment.